.github/workflows/test-monorepo.yml

```diff
@@ -18,3 +18,20 @@ jobs:
           yarn install --immutable
           yarn plugin import workspace-tools
           ../action-npm-publish/scripts/main.sh
+  checkout_publish_skunkworks_dry_run_skip:
```

Comment:
Would you mind adding a comment here and the one before this which describes what this is testing? I think that would help us in the future as right now it's not totally clear.

Reply:
i added notes and updated this to remove the `_dry_run_skip` since that was also confusing (and wrong)